ibazel/command/default_command_test.go

```diff
@@ -60,7 +60,7 @@ func TestDefaultCommand(t *testing.T) {
 	}
 
 	// This is synonymous with killing the job so use it to kill the job and test everything.
-	c.NotifyOfChanges(nil)
+	c.AfterRebuild(nil)
```

Comment:
Looks like this test should probably be calling `BeforeRebuild` (based on the comment on the line above).

Reply:
I think having both BeforeRebuild and AfterRebuild restores the intended functionality of this test. I implemented it and force pushed